build-plugin.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+#
+# Building migrations with datastore plugin
```

Comment:
Is any of this datastore specific, or just arbitrary plugins (e.g. could IPLD plugins work too?)

Reply:
I assumed this was only for datastore plugins, but I guess it could be built with any.  This is just for building a filestore repo migration, so does it even make sense to support _any_ plugin?  I do not know.

Do I need to add another argument, or prompt,  for the name of the plugin?